jenkins/jobs/lxc-build-android.yaml

```diff
@@ -0,0 +1,30 @@
+- job:
+    name: lxc-build-android
+    concurrent: true
+    description: Builds an android tarball from the provided source.
+    node: lxc-priv && amd64
+    project-type: freestyle
+
+    properties:
+      - build-discarder:
+          num-to-keep: 15
+
+    parameters:
+      - string:
+          name: sha1
+          description: Hash of the commit to build
+
+      - string:
+          name: branch
+          description: Source git branch
+
+    builders:
```

Comment:
Small nit but in the existing jobs, this goes to the top, right after project-type and before properties.

Reply:
Done